cav_srvs/srv/ComputePlanCost.srv

```diff
@@ -0,0 +1,15 @@
+#
+# ComputePlanCost.srv
+# 
+# Provided by the cost plugin node. Returns a final cost of a given maneuver plan.
+#
+# @author Fang Zhou
+# @version 0.1
+#
+
+# Request
+cav_msgs/ManeuverPlan maneuver_plan
+
+---
+# Response
+string plan_cost

```

Comment:
Not sure how to interpret this string, is the Costplugin node actually returning a string? I kind of anticipated that this would be a float or a double. How is this string formatted?

Reply:
This should be a double. Thanks for pointing out. Will change it now.